opentelemetry-otlp/src/exporter/tonic/logs.rs

```diff
@@ -58,35 +58,40 @@ impl TonicLogsClient {
 
 #[async_trait]
```

Comment:
we can remove async_trait from everywhere now.

Reply:
This is done, thanks.